tests/common/BundlerTest.cs

```diff
@@ -0,0 +1,143 @@
+// Tests are common to both mtouch and mmp
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+
+using NUnit.Framework;
+
+using Mono.Cecil;
+
+using Xamarin.Tests;
+
+#if MONOTOUCH
+using BundlerTool = Xamarin.MTouchTool;
+#else
+using BundlerTool = Xamarin.MmpTool;
+#endif
+
+namespace Xamarin
+{
+	[TestFixture]
+	public class BundlerTests
+	{
+		[Test]
+#if __MACOS__
+		[TestCase (Profile.macOSMobile)]
```

Comment:
Do we need/want Full here?

Reply:
I don't think so, because this is not testing any API/code inside the platform assembly, only mmp (the linker), which should not depend on the profile (note that for XI I'm not running the test for tvOS/watchOS, only iOS).

Reply:
That's fair.